docs/src/prototypes/chatMessageWithPopover/Popover.tsx

```diff
@@ -36,19 +36,40 @@ class Popover extends React.Component<PopoverProps, PopoverState> {
     opacity: 0,
 
     '& .smile-emoji': {
-      display: 'none',
+      position: 'absolute',
+      opacity: 0,
+      zIndex: -1,
     },
 
     '&.focused .smile-emoji': {
-      display: 'flex',
+      position: 'initial',
+      zIndex: 'initial',
+      opacity: 1,
     },
 
     '&:hover .smile-emoji': {
-      display: 'flex',
+      position: 'initial',
+      zIndex: 'initial',
+      opacity: 1,
     },
   })
 
   render() {
+    const overridenToolbarBehavior: Accessibility = (props: any) => {
+      const toolbarFocusZone = toolbarBehavior(props).focusZone
```

Comment:
This can be simplified to:
```
const behavior = toolbarBehavior(props)
behavior.focusZone.props.defaultTabbableElement = {...}
return behavior
```

Reply:
looks better, thx